src/mscorlib/src/Internal/Runtime/Augments/EnvironmentAugments.cs

```diff
@@ -17,5 +18,11 @@ public static class EnvironmentAugments
         public static bool HasShutdownStarted => Environment.HasShutdownStarted;
         public static string StackTrace => Environment.StackTrace;
         public static int TickCount => Environment.TickCount;
+        public static string GetEnvironmentVariable(string variable) => Environment.GetEnvironmentVariableCore(variable);
+        public static string GetEnvironmentVariable(string variable, EnvironmentVariableTarget target) => Environment.GetEnvironmentVariableCore(variable, target);
```

Comment:
Thinking about it some more ... I think it may be better to keep it as you have it. It may safe us from having one private copy of registry. The registry is needed in CoreLib for other reasons, so it does not hurt to leverage for the environment stuff as well.